player/lua/select.lua

```diff
@@ -301,11 +270,14 @@ mp.add_key_binding(nil, "select-subtitle-line", function ()
     local duration = mp.get_property_native("duration", math.huge)
 
     -- Strip HTML and ASS tags.
-    for line in r.stdout:gsub("<.->", ""):gsub("{\\.-}", ""):gmatch("[^\n]+") do
+    for line in data:gsub("<.->", ""):gsub("{\\.-}", ""):gmatch("[^\n]+") do
         -- ffmpeg outputs LRCs with minutes > 60 instead of adding hours.
-        sub_times[#sub_times + 1] = line:match("%d+") * 60 + line:match(":([%d%.]*)")
-        sub_lines[#sub_lines + 1] = format_time(sub_times[#sub_times], duration) ..
-                                    " " .. line:gsub(".*]", "", 1)
+        local sub_line = line:gsub(".*]", "", 1)
```

Comment:
This was already broken but I noticed that `('[foo][bar]baz'):gsub(".*]", "", 1)` returns `baz`. You can update to it `line:gsub(".-]", "", 1)` while at it.

Reply:
Identified additional issues and further refactored the parsing of subtitle content